lnrpc/routerrpc/router_server.go

```diff
@@ -321,9 +321,14 @@ func (s *Server) SendPaymentV2(req *SendPaymentRequest,
 	err = s.cfg.Router.SendPaymentAsync(payment)
 	if err != nil {
 		// Transform user errors to grpc code.
-		if err == channeldb.ErrPaymentInFlight ||
-			err == channeldb.ErrAlreadyPaid {
+		switch {
+		case errors.Is(err, channeldb.ErrPaymentExits):
+			fallthrough
```

Comment:
hahah sure np. And I'm curious about the "not a big fan of fallthroughs" part, how so?

Reply:
I think this is also partially an artifact of the `errors.Is` linter. Previously this could be a simple list of cases: `case A, B, C:`. 

I doubt the value of this linter. A lot of errors aren't wrapped, but do require this verbose and potentially error prone matching.

Reply:
> And I'm curious about the "not a big fan of fallthroughs" part, how so?

I always need to refresh my memory about whether `fallthrough` means "continue evaluating the next `case` statement" vs. "execute the code of the next `case` block even if the condition of that `case` statement doesn't evaluate to `true`. Probably just a remnant of me coding Java for too long (where the `switch` works quite differently). So feel free to ignore my comment.

Reply:
My take is I can now not think about whether using `==` or `errors.Is` but only use `errors.Is` since it works for both case. I think go doc also recommends it. But yeah a list of cases can work, switched.